tests/Test/Pager/Subscriber/Filtration/Doctrine/ORM/QueryTest.php

```diff
@@ -37,20 +37,20 @@ public function shouldHandleApcQueryCache(): void
         ];
 
         $em = \Doctrine\ORM\EntityManager::create($conn, $config);
-        $schema = array_map(function ($class) use ($em) {
+        $schema = array_map(static function ($class) use ($em) {
```

Comment:
Why `static` ?

Reply:
Because function doesn't need to access `$this`. It's a micro-optimization.